src/main/java/direction/Point.java

```diff
@@ -0,0 +1,18 @@
+package direction;
+
+public record Point(
```

Comment:
```suggestion
public record Point(int x, int y) {
```
한 줄로 쓸 수 있는걸 세 줄로 늘릴 필요가 있을까요?

Reply:
맞아요.. 제가 컨벤션을 제대로 지키지 않은 것 같습니다.

[51ee0bb](https://github.com/woowacourse/java-janggi/pull/72/commits/51ee0bb89c387349332c342f8b7dc00070fe4a29)